manager/controllers/app/m4dapplication_controller.go

```diff
@@ -49,10 +52,10 @@ type M4DApplicationReconciler struct {
 func (r *M4DApplicationReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
```

Comment:
Comment says it generates a Blueprint CRD.  Need to add that it creates either a blueprint for single cluster or a plotter for multi-cluster

Reply:
Updated comments.